project/ProjectPlugin.scala

```diff
@@ -38,7 +38,7 @@ object ProjectPlugin extends AutoPlugin {
       val fs2: String                = "2.2.1"
       val fs2Grpc: String            = "0.6.0"
       val fs2Kafka: String           = "0.20.2"
-      val grpc: String               = "1.24.2"
+      val grpc: String               = "1.26.0"
```

Comment:
I was taking a look to this yesterday, I'm not sure if we should wait until the next `fs2-grpc` release

https://mvnrepository.com/artifact/org.lyranthe.fs2-grpc/java-runtime_2.13/0.6.0

Reply:
also I'm not sure about 1.26.0, it's still not mentioned on https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty

Reply:
@fedefernandez yeah, makes sense.

@BenFradet seems like 1.25.x and 1.26.x are using the same netty dependencies: https://github.com/grpc/grpc-java/blob/v1.26.0/build.gradle#L147